ServerTCPWpfApp/MainWindow.xaml.cs

```diff
@@ -12,13 +13,13 @@ namespace ServerTCPWpfApp;
 /// </summary>
 public partial class MainWindow : Window
 {
-    private const int Port = 8910;
-    private const int MaxConnections = 16;
-    private const int BufferSize = 256;
-    private static readonly IPHostEntry IpHost = Dns.GetHostEntry("localhost");
-    private readonly IPAddress _ipAddress = IpHost.AddressList[0];
-    private IPEndPoint? _iPEndPoint;
-    private Socket? _listenSocket;
+    private const int _port = 8910;
+    private readonly NewsServer _server = new();
+
+    public Dictionary<string, NetworkClient> Clients
```

Comment:
Simplify to `public Dictionary<string, NetworkClient> Clients => _server.Clients;`

Reply:
It was replaced with `_server.Clients`.